### DISCUSSION TITLE
Issue with negative dimensions in CNN hyperparameter optimization

### DISCUSSION THREAD

**Discussion by CaueSousa** [optuna/optuna#4922] [Q&A]
Dear community, hello!
I'm trying to use Optuna to optimize various hyperparameters of a convolutional block; however, some specific combinations of parameters suggested by Optuna result in the output dimension becoming negative.
Do you know of any way to eliminate combinations that lead to absurd results?

**Comment by nzw0301** [comment]
Hi, could you share minimal reproducible codes with us?
